### PR TITLE
Fix missing Binary Info for boot_stage2

### PR DESCRIPTION
**Pull request by kilograham**
Regression in SDK2.0.0

### CODE REVIEW

**Comment by kilograham** [comment]
Fixes #1842

File: src/rp2350/boot_stage2/CMakeLists.txt [hunk 1]
```diff
@@ -63,12 +63,13 @@ function(pico_define_boot_stage2 NAME SOURCES)
     find_package (Python3 REQUIRED COMPONENTS Interpreter)
 
     add_custom_target(${NAME}_bin DEPENDS ${ORIGINAL_BIN})
-    add_custom_command(OUTPUT ${ORIGINAL_BIN} DEPENDS ${NAME} COMMAND ${CMAKE_OBJCOPY} -Obinary $<TARGET_FILE:${NAME}> ${ORIGINAL_BIN})
+    add_custom_command(OUTPUT ${ORIGINAL_BIN} DEPENDS ${NAME} COMMAND ${CMAKE_OBJCOPY} -Obinary $<TARGET_FILE:${NAME}> ${ORIGINAL_BIN}
+        VERBATIM)
 
     add_custom_target(${NAME}_padded_checksummed_asm DEPENDS ${PADDED_CHECKSUMMED_ASM})
     add_custom_command(OUTPUT ${PADDED_CHECKSUMMED_ASM} DEPENDS ${ORIGINAL_BIN}
-            COMMAND ${Python3_EXECUTABLE} ${PICO_BOOT_STAGE2_DIR}/pad_checksum -s 0xffffffff -a $<IF:${PICO_RISCV},riscv,arm> ${ORIGINAL_BIN} ${PADDED_CHECKSUMMED_ASM}
```
Comment (lurch) [inline src/rp2350/boot_stage2/CMakeLists.txt:70]: Is the removal of the `-a` flag here deliberate? It looks like https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2350/boot_stage2/pad_checksum _does_ use that flag? (but I dunno enough of the the low-level details to know whether that's significant or not!)

Comment (will-v-pi) [reply src/rp2350/boot_stage2/CMakeLists.txt:70]: Yes, it shouldn't be removed here, as it is necessary on RP2350

**Review by will-v-pi — COMMENTED** [review]
Other than @lurch's comment about the `-a` flag, this all looks and works fine

File: src/rp2350/boot_stage2/CMakeLists.txt (hunk 1 above)
Comment (kilograham) [reply src/rp2350/boot_stage2/CMakeLists.txt:70]: good catch - COVID development again!

**Pull request merged**